lib/dust.js

```diff
@@ -1025,11 +1074,8 @@
           if(ended) {
             return;
           }
-          if(errorBody) {
-            chunk.render(errorBody, context.push(err));
-          } else {
-            dust.log('Unhandled stream error in `' + context.getTemplateName() + '`', INFO);
-          }
+          chunk.renderError(err, context, bodies);
+          dust.log('Unhandled stream error in `' + context.getTemplateName() + '`', INFO);
```

Comment:
Note that this is a logic difference.  Previously `dust.log(...)` was only called when there was no render body whereas now it is always logged.  I think it's ok.